libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -944,7 +991,7 @@ bool ofDirectory::create(bool recursive){
 
 //------------------------------------------------------------------------------------------------------------
 bool ofDirectory::exists() const {
-	return std::filesystem::exists(myDir);
+	return (myDir == "" || std::filesystem::exists(myDir));
```

Comment:
I had to add this as ofDirectory::exists was failing when it was set to "".  
In the case of a directory it does exist if the path is "" as it is the cwd. 

So things like:

```
ofDirectory dir; 
dir.openFromCWD( ofFilePath::getEnclosingDirectory("test1.txt") ); 
dir.exists();
```
were failing. 

I think we didn't notice this in the past as a directory was never "" as we were always adding ofToDataPath to everything.